src/main/java/org/gelecekbilimde/scienceplatform/common/util/validation/UsernameValidator.java

```diff
@@ -23,6 +23,13 @@ public boolean isValid(String value, ConstraintValidatorContext context) {
 			return false;
 		}
 
+		if (value.length() <= 3 || value.length() >= 20) {
+			context.disableDefaultConstraintViolation();
+			context.buildConstraintViolationWithTemplate("Username must be between 3 and 20 characters long")
+				.addConstraintViolation();
+			return false;
+		}
+
 		if (!lowerCasedValue.matches(USERNAME_REGEX)) {
 			context.disableDefaultConstraintViolation();
 			context.buildConstraintViolationWithTemplate("Username must be alphanumeric")
```

Comment:
```suggestion
			context.buildConstraintViolationWithTemplate("must be alphanumeric")
```

Reply:
@agitrubard approve lar mısın artık  🥲

Reply:
@MenekseYuncu commiti düzgün bir şekilde atar mısın? `...` şeklinde olmuş. Merge commit olarak mergelediğimiz için çok doğru gibi görünüyor
![Uploading image.png…]()

Reply:
@agitrubard  düzeltildi